src/test/java/stashpullrequestbuilder/stashpullrequestbuilder/StashBuildTriggerTest.java

```diff
@@ -0,0 +1,185 @@
+package stashpullrequestbuilder.stashpullrequestbuilder;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
```

Comment:
minor: I would prefer we don't use wildcard imports, most IDEs have the option to expand them

Reply:
Done. It's common to use wildcards for matchers in unit tests, but it's not a style requirement, so I'm fine either way.

Reply:
I plan to apply google style guide to this someday, it will be less changes if we don't use wildcards